modules/redisearch.ts

```diff
@@ -923,7 +1117,7 @@ export type FTConfig = {
 /**
  * The info response
  */
-export type FTInfo = {
+export interface FTInfo {
```

Comment:
What's the benefit of this being an interface rather than a type?

Reply:
https://github.com/danitseitlin/redis-modules-sdk-ts/pull/136#discussion_r698265604